embedding/mbedk64f/.gitignore

```diff
@@ -0,0 +1,5 @@
+libjerry/
```

Comment:
this folder is where jerry libraries are copied, so I think it won't be necessary to include to repo